packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -326,39 +360,86 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly IAM_ROLES_ANYWHERE = new InterfaceVpcEndpointAwsService('rolesanywhere');
   public static readonly INSPECTOR = new InterfaceVpcEndpointAwsService('inspector2');
   public static readonly IOT_CORE = new InterfaceVpcEndpointAwsService('iot.data');
+  public static readonly IOT_LORAWAN = new InterfaceVpcEndpointAwsService('iotwireless.api');
```

Comment:
This one doesn't seem to match up. Is this correct?

Reply:
@TheRealAmazonKendra 
I too am struggling with the naming of it. The basis for this decision is twofold.

1. the line of the following page searched for "'iotwireless.api'"
https://docs.aws.amazon.com/ja_jp/vpc/latest/privatelink/integrated-services-vpce-list.html

2. the official documentation from a Google search for "iotwireless.api"
https://docs.aws.amazon.com/ja_jp/iot/latest/developerguide/connect-iot-lorawan-onboard-control-plane-endpoint.html

Reply:
Hmm. Maybe this to keep it consistent with documentation?
```suggestion
  public static readonly IOT_CORE_FOR_LORAWAN = new InterfaceVpcEndpointAwsService('iotwireless.api');
```